src/tests/unit/userService/updateRefreshToken.test.ts

```diff
@@ -0,0 +1,26 @@
+import { mockId, mockToken, saveUser } from '../../mocks/mockUser';
+import UserService from '../../../services/userService';
+import { User } from '../../../models/userModel';
+import { setupTests } from '../../setupTests';
+
+const userService = new UserService();
+
+setupTests('updateRefreshToken', () => {
+  test('returns the user with an old token', async () => {
+    const newUser = await saveUser();
+    const user = await userService.updateRefreshToken(newUser.id, mockToken);
+    expect(user?.refreshToken).toBe(newUser.refreshToken);
+  });
```

Comment:
To be honest that function could be void, since I'm not using that return anywhere. By default findByIdAndUpdate() returns value before the update, so that's what my function returns. I guess this test is unecessary. Should I just remove it or also change the function to not return anything?

Reply:
@Kirosana15 Here I'm more talking about the function logic than the return values.

If I understand this test correctly, when we update the refresh token for a specific user, the refresh token stays the same, all the time.

My concern is that the refresh token is not "refreshed" here, it stays the same as it was before the update (that's what I assume based on the test name and the assertion inside).
I believe "updating refresh token" should indeed change the refresh token 🤔 

---

When it comes to the return value, it's up to you. If you don't need the returned value, it's valid to change the signature to `void`